### ISSUE TITLE
error: failed to run custom build command for `ring v0.16.20`

### ISSUE THREAD

**Issue by harsh4786** [project-serum/anchor#563]
![image](https://user-images.githubusercontent.com/50767810/127646399-5b49510f-362d-4766-9ed6-ec6586bdf37c.png)
![Screenshot 2021-07-30 112645](https://user-images.githubusercontent.com/50767810/127646552-6a6baaa8-9406-4b91-a562-64a99a75265c.jpg)
i tried running the command "cargo install --git https://github.com/project-serum/anchor --tag v0.11.1 anchor-cli --locked" and i got the given error

**Comment by fanatid** [comment]
Try to use `x86_64-pc-windows-msvc` target.
https://github.com/briansmith/ring/issues?q=aes_nohw.c

**Comment by gyandata** [comment]
@harsh4786 are you able to resolve this issue?

**Comment by paul-schaaf** [comment]
not an issue with anchor so Im closing this

**Comment by harsh4786** [comment]
> @harsh4786 are you able to resolve this issue?

Yeah not an anchor issue, was something wrong with my Windows installation back then, I switched to Ubuntu, try ubuntu ser
